doc/samples/custom_loader.py

```diff
@@ -3,21 +3,23 @@
 import sys
 
 from doit.task import dict_to_task
-from doit.cmd_base import TaskLoader
+from doit.cmd_base import TaskLoader2
 from doit.doit_cmd import DoitMain
 
 my_builtin_task = {
     'name': 'sample_task',
     'actions': ['echo hello from built in'],
     'doc': 'sample doc',
-    }
+}
 
-class MyLoader(TaskLoader):
-    @staticmethod
-    def load_tasks(cmd, opt_values, pos_args):
+
+class MyLoader(TaskLoader2):
```

Comment:
I think better inlcude ``setup()`` in the example for documentation purpose. Even if just ``pass``...